valhalla/mjolnir/directededgebuilder.h

```diff
@@ -30,6 +30,8 @@ class DirectedEdgeBuilder : public baldr::DirectedEdge {
    * @param  restrictions   Mask of simple turn restrictions at the end node
    *                        of this directed edge.
    * @param  bike_network   Mask of bike_networks from relations.
+   * @param  reclass_ferry  Reclassify ferry boolean; Added for
+   * https://github.com/valhalla/valhalla/issues/1889
```

Comment:
it would be preferable to explain what this is doing as opposed to linking a github issue